### ISSUE TITLE
Deploy to open-vsx

### ISSUE THREAD

**Issue by StevenvdSchoot** [disroop/vs-code-conan#128]
Currently this extension is only available in the Microsoft's Extension Marketplace. Would is be possible to make it available on [open-vsx](https://open-vsx.org/) as well? This allows users of VSCode alternatives (like VSCodium and Eclipse Theia) to use this extensions as well.

**Comment by mymichu** [comment]
Yes, I will have a look into this and deploy it this week to those platforms. I will let you know when the plugin is available.
